app/src/components/searchWidget/Autocomplete.tsx

```diff
@@ -76,16 +76,16 @@ class Autocomplete extends React.Component<AutoCompleteProps, any> {
     }
 
     onSelectChanged(value: any, action: ActionMeta) {
-        this.props.onChange(this.props.defaultValue);
```

Comment:
I'm not sure that correct value will be passed there. check if your value is "MEL" or "SYD" string

Reply:
I had changed it to value, it is working